src/v3/src/components/InputText/InputText.tsx

```diff
@@ -17,7 +17,6 @@ import {
   InputLabel,
   Typography,
 } from '@okta/odyssey-react-mui';
-import { h } from 'preact';
```

Comment:
Was this removed on purpose? I think this is added to any file that contains jsx

Reply:
Oh oops, will add back I think I thought at one point it was giving me a lint error but maybe I was mistaken